### DISCUSSION TITLE
Tab position when opening url from Vomnibar

### DISCUSSION THREAD

**Discussion by dpetka2001** [gdh1995/vimium-c#683] [Q&A] [answered]
Hello, I would like to know if there is a way/option to choose where the new tab would be positioned.
For example when i press 'O' to open url in new tab from Vomnibar, i would like the new tab to be positioned last instead
of next to the current tab like it is configured by default.

**Comment by gdh1995** [comment, marked as answer]
Please try:
```
map O Vomnibar.activateUrlInNewTab position="end"
```

In a next version I'll add `position=...` to the `actions` field in `Vomnibar settings`.

**Reply by dpetka2001** [reply]
Thank you very much for your response. I used `activateInNewTab` instead to make it work how i want.
